src/bridge.ts

```diff
@@ -65,13 +66,18 @@ class DenoBridge {
     // a malformed semver range. If this does happen, let's throw an error so
     // that the tests catch it.
     if (downloadedVersion === undefined) {
-      throw new Error('Could not read downloaded binary')
+      if (this.onAfterDownload) {
+        this.onAfterDownload(true)
+      }
+      throw new Error(
```

Comment:
I don't think we should mention CLI here. We're in Edge Bundler, which is also a dependency of Netlify Build, so it could be used in a context that doesn't involve the CLI at all.